Touchup/travis migrate (#4742)
circleci/circleci-docs: jekyll/_cci2/migrating-from-travis.md

This document provides an overview of how to migrate from Travis CI to CircleCI.

The example build configurations referenced throughout this article are based
on this [example JavaScript repository](https://github.com/CircleCI-Public/circleci-demo-javascript-express/blob/master/.circleci/config.yml).

## Prerequisites

This document assumes that:

> 1. You have an account with CircleCI that is linked to a repository. If you don't yet,

Reviewer: ```suggestion
1. You have an account with CircleCI that is linked to a repository. If you do not have an account,
```

consider reading our [Getting Started Guide]({{ site.baseurl }}/2.0/getting-started/).
1. You understand the [Basic Concepts]({{ site.baseurl }}/2.0/concepts/) in CircleCI.

## Why Migrate to CircleCI?

> - **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on an [annual plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.

Reviewer: ```suggestion
- **Scaling Concurrency**: You can run up to 80 concurrent jobs on our monthly Performance Plan or even more on a [custom plan](https://circleci.com/pricing/). Travis CI has capped concurrencies of 1, 2, 5, and 10 on each of their plans.
```
I would refer to our "annual" plan as "custom" as that is the name of the plan (that happens to have a customizable annual billing cycle))

- **Orbs**: Rather than proprietary integrations, CircleCI offers [orbs]({{ site.baseurl }}/2.0/orb-intro/), which are reusable, templated configuration. On top of connecting to services and tools, orbs can be used to standardize and templatize configuration for your team and organization as well. [Visit the registry](https://circleci.com/orbs/registry/).

## Configuration Files

Both Travis CI and CircleCI make use of a _configuration file_ to define your 
workflows and jobs. The only difference is that your CircleCI configuration
will live in `.circleci/config.yml` at the root of your repository.

Below, you'll find a side-by-side comparison of different configuration declarations.

| Travis CI         | Circle CI                                                                          | Description                                                                                          |
|-------------------|------------------------------------------------------------------------------------|------------------------------------------------------------------------------------------------------|

> | language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language. Instead, CircleCI allows you to use any of the major operating systems or [Docker images]({{ site.baseurl }}/2.0/circleci-images/#next-gen-language-images) as executors. Specify a Docker image with your needed prerequisites or use a VM and install other needed libraries (see pre-installed software for [Windows]({{ site.baseurl }}/2.0/hello-world-windows/), [macOS]({{ site.baseurl }}/2.0/testing-ios/#supported-xcode-versions), and [Linux]({{ site.baseurl }}/2.0/docker-to-machine/#pre-installed-software) images). Then, use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |

Reviewer: ```suggestion
| language:,<br>os: | [docker, machine, macos, windows]({{ site.baseurl }}/2.0/executor-types/)          | CircleCI doesn't assume dependencies or commands based on a language; instead, choose an executor and use `run:` steps as shown below to execute commands you require (e.g., install, build, test). |
```

I think this line should be simplified. The information about pre-installed software and the links to other documents should all be found in the already linked to document on executor-types (which can be done in another PR). We should aim to keep table elements from getting too text-heavy (both for contributors to the docs and readers).

cache: npm
```

> For basic builds, a Travis CI configuration will leverage a language's best known

Reviewer: Good catch.
